src/main/cpp/subsystems/ShooterWheelsSubsystem.cpp

```diff
@@ -6,8 +6,8 @@
 
 ShooterWheelsSubsystem::ShooterWheelsSubsystem()
 {
-    TopShooterPID.SetP(0.000015);
-    BottomShooterPID.SetP(0.000015);
+    TopShooterPID.SetP(0.000005);
```

Comment:
These could possibly get updated straight to the motor controllers if we wanted but isn't a priority